python/apps/taiga/tests/unit/taiga/stories/stories/test_services.py

```diff
@@ -706,6 +709,7 @@ async def test_reorder_stories_ok():
         )
         fake_stories_events.emit_when_stories_are_reordered.assert_awaited_once()
         assert fake_notifications.notify_when_story_status_change.await_count == 3
+        assert fake_notifications.notify_when_story_workflow_change.assert_awaited()
```

Comment:
As I see it it's normal this test to fail remotely (and also to me locally) , as reordering stories doesn't call the update_story method (and this is the only point where you launch the notifications you're awaiting).

I don't understand why this test pass on your machine.